docs/basics/RootSaga.md

```diff
@@ -0,0 +1,84 @@
+# Setting up your root Saga
+
+In the beginner tutorial it is shown that your root saga will look something like this
+
+```javascript
+export default function* rootSaga() {
+  yield all([
+    helloSaga(),
```

Comment:
Not sure if mentioning yielding iterators (`helloSaga()`'s result in this case) in yet another place is a good thing, while those are technically handled by redux-saga I would restrain from using it over call/fork which should be the recommended way.

Reply:
Well the context here is to reference the root saga implementation in the tutorial, just as a jumping off point. I could use stronger language to suggest that this way is is *not* recommended, but it seems necessary to include atleast some type of nod to the tutorial's root saga.